lib/handlers/get.js

```diff
@@ -103,22 +106,46 @@ async function handler (req, res, next) {
       debug('   sending data browser file: ' + dataBrowserPath)
       res.sendFile(dataBrowserPath)
       return
-    } else if (stream) {
+    } else if (stream) { // EXIT text/html
       res.setHeader('Content-Type', contentType)
       return stream.pipe(res)
     }
   }
 
   // If request accepts the content-type we found
+  // if (stream && negotiator.mediaType([contentType])) {
+  //   res.setHeader('Content-Type', contentType)
+  //   if (contentRange) {
+  //     const headers = { 'Content-Range': contentRange, 'Accept-Ranges': 'bytes', 'Content-Length': chunksize }
+  //     res.writeHead(206, headers)
+  //     return stream.pipe(res)
+  //   } else {
+  //     return stream.pipe(res)
+  //   }
+  // }
+
```

Comment:
Just checking—should this commented-out section remain here, or is it safe to remove?

Reply:
Thanks, please remove